### DISCUSSION TITLE
Access an Assets resource (Android)

### DISCUSSION THREAD

**Discussion by davidec** [unoplatform/uno#10409] [Q&A]
Hi,
I start to use UNO. (Last version Net6)
I put an image in folder Assets in share project and it show correctly in my page (Android)

`<Image Source="ms-appx:///Assets/btnp_email.png"`

It is work, I thinks that the resource (the image) is successfully copied to the Android in some folder.

Now I would like to try to access the file runtime but I don't find a solution or an example code

This is a simple PNG file but i will work with a sqlite file and I need to access runtime 
(In a xamarin Forms project I copied this file in Assets folder of Android project with AndroidAssets and Copy Always)
Thanks

**Comment by jeromelaban** [comment]
Hi, `ms-appx` assets are not accessible from the standard file system APIs and must be accessed through [`StorageFile.GetFileFromApplicationUriAsync`](https://platform.uno/docs/articles/features/file-management.html?q=storage#support-for-storagefilegetfilefromapplicationuriasync). The returned `IStorageFile` will provide you a way to copy the content and use as needed.
